genet/utils/elevation.py

```diff
@@ -60,3 +62,22 @@ def validation_report_for_node_elevation(elev_dict, low_limit=-50, mont_blanc_he
                    'extremely_low_values_dict': too_low}}
 
     return report
+
+
+def write_slope_xml(links, output_dir):
```

Comment:
Hm could do, but to me it makes sense to have it with other elevation stuff, since it only gets used for elevation? The tests for elevation are all in one place then, and the fixtures are there...

Reply:
so i'm thinking this method should be generalised to just 'object attributes' - as per matsim's intention for this file type, but not in this PR. I guess the generalisation work could also involve the move though